lib/bots/versionbot.ts

```diff
@@ -845,15 +892,56 @@ export class VersionBot extends GithubBot.GithubBot {
                                 };
                                 this.flowdock.postToInbox(flowdockMessage);
                             }
-                            this.log(ProcBot.LogLevel.DEBUG, `MergePR: Merged ${owner}/${repo}#${prInfo.number}`);
+                            this.log(ProcBot.LogLevel.INFO, `MergePR: Merged ${owner}/${repo}#${prInfo.number}`);
                         });
                     }
                 });
             }
         });
     }
 
-    private reportError(error: VersionBotError) {
+    private validMaintainer(config: VersionBotConfiguration, event: GithubBotApiTypes.PullRequestEvent): void {
```

Comment:
nit: this method doesn't have a verb in the name. `ensureEventIsFromMaintainer`? `verifyEventSender`?